.github/workflows/images-latest.yml

```diff
@@ -51,34 +51,32 @@ jobs:
           - { name: python, ecosystem: pip }
           - { name: terraform, ecosystem: terraform }
     env:
-      TAG: ${{ github.sha }}
+      COMMIT_SHA: ${{ github.sha }}
       NAME: ${{ matrix.suite.name }}
       ECOSYSTEM: ${{ matrix.suite.ecosystem }}
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
 
-      - name: Build dependabot-updater image
+      - name: Build the dependabot-updater-<ecosystem> image
```

Comment:
Unfortunately step names don't have access to the env vars, otherwise I'd have used `dependabot-updater-${ECOSYSTEM}` in the name... I tried. 😄